doc/Security.xml

```diff
@@ -947,13 +995,311 @@ Added certificate-based client authentication</revremark>
+      <para>JWTs consists of three elements:</para>
+      <para>
+        <itemizedlist>
+          <listitem>
+            <para>JOSE header</para>
+          </listitem>
+          <listitem>
+            <para>Payload</para>
+          </listitem>
+          <listitem>
+            <para>Signature</para>
+          </listitem>
+        </itemizedlist>
+      </para>
+      <para>The JOSE header shall declare that the encoded object is a JSON Web Token by setting the <emphasis>typ</emphasis> claim to <emphasis>JWT</emphasis>, and the JWT is a JWS that is signed using the asymmetric algorithm identified by the <emphasis>alg</emphasis> claim with a value defined in RFC7518. Since the keystore does not support symmetric encryption, symmetric keyed hashing algorithms shall not be used.</para>
+      <para>The JWT payload shall include the following standard claims:</para>
+      <para>
+        <itemizedlist>
+          <listitem>
+            <para><emphasis>iss</emphasis>: Issuer, the URI of the server that generated the JWT.</para>
+          </listitem>
+          <listitem>
+            <para><emphasis>aud</emphasis>: Audience, a list of strings representing the recipients that the JWT is intended for. Each principal intended to process the JWT shall identify itself with a value in the audience claim. If the principal processing the claim does not identify itself with a value in the audience claim when this claim is present, then the JWT shall be rejected.</para>
+          </listitem>
+          <listitem>
+            <para><emphasis>exp</emphasis>: Expiration Time.</para>
+          </listitem>
+          <listitem>
+            <para><emphasis>nbf</emphasis>: Not before.</para>
+          </listitem>
+        </itemizedlist>
+      </para>
+      <para>For the <emphasis>exp</emphasis>, <emphasis>nbf</emphasis> claims, a device shall reject a token when the current time is not within the range of claims nbf and exp . It shall reject a token when at least one of the claims is missing.</para>
+      <para>The JWT payload shall include the <emphasis>roles</emphasis> claim, as defined within [RFC 7643]:</para>
+      <para>
+        <itemizedlist>
+          <listitem>
+            <para><emphasis>roles</emphasis>: Access class. One of the authenticated classes of the default access policy prefixed with the string <emphasis>onvif:</emphasis>, i.e. "onvif:Administrator", "onvif:Operator" or "onvif:User" as defined also within the ONVIF Core Specification. This access level will be used to authorize access to the required function.</para>
+          </listitem>
+        </itemizedlist>
+      </para>
+      <para>The signature is evaluated by applying ECDSA using secp256r1 and SHA-256 hashing to the base64-encoded header and payload, concatenated by a '.'</para>
```

Comment:
"The signature is evaluated " Is there an RFC to reference to? This text is just informative.